There is an error that can be related with this PR:

```
2014-08-12 13:08:08,636 23125 ERROR openerp_test openerp.tools.safe_eval: Cannot eval u'test_exceptions()'
Traceback (most recent call last):
File "/home/travis/build/OCA/odoo-7.0/openerp/tools/safe_eval.py", line 286, in safe_eval
return eval(test_expr(expr, _SAFE_OPCODES, mode=mode), globals_dict, locals_dict)
File "", line 1, in <module>
File "/home/travis/build/OCA/odoo-7.0/openerp/osv/orm.py", line 377, in function_proxy
return attr(self._cr, self._uid, [self._id], *args, **kwargs)
File "/home/travis/build/OCA/sale-workflow/sale_exceptions/sale.py", line 146, in test_exceptions
exception_ids = self.detect_exceptions(cr, uid, ids, context=context)
File "/home/travis/build/OCA/sale-workflow/sale_exceptions/sale.py", line 171, in detect_exceptions
current_exception_ids = set(exc.id for exc in order.exception_ids)
File "/home/travis/build/OCA/odoo-7.0/openerp/osv/orm.py", line 504, in __getattr__
raise AttributeError(e)
AttributeError: "Field 'exception_ids' does not exist in object 'browse_record(sale.order, 1)'"
```

Can you please check it?

--- a/sale_exceptions/sale.py
+++ b/sale_exceptions/sale.py
@@ -168,7 +168,7 @@ class sale_order(orm.Model):
         for order in self.browse(cr, uid, ids, context=context):
             if order.ignore_exceptions:
                 continue
-            current_exception_ids = set(exc.id for exc in order.exception_ids)
+            current_exception_ids = set(exc.id for exc in order.exceptions_ids)
             exception_ids = self._detect_exceptions(cr, uid,
                                                     order,
                                                     order_exceptions,

